github/copilot.go

```diff
@@ -0,0 +1,263 @@
+// Copyright 2023 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+)
+
+// CopilotService provides access to the Copilot-related functions
+// in the GitHub API.
+//
+// GitHub API docs: https://docs.github.com/en/rest/copilot/
+type CopilotService service
+
+// OrganizationCopilotDetails represents the details of an organization's Copilot for Business supbscription.
+type OrganizationCopilotDetails struct {
+	SeatBreakdown         *SeatBreakdown `json:"seat_breakdown"`
+	PublicCodeSuggestions string         `json:"public_code_suggestions"`
+	CopilotChat           string         `json:"copilot_chat"`
+	SeatManagementSetting string         `json:"seat_management_setting"`
+}
+
+// SeatBreakdown represents the breakdown of Copilot for Business seats for the organization.
+type SeatBreakdown struct {
+	Total               int64 `json:"total"`
+	AddedThisCycle      int64 `json:"added_this_cycle"`
+	PendingCancellation int64 `json:"pending_cancellation"`
+	PendingInvitation   int64 `json:"pending_invitation"`
+	ActiveThisCycle     int64 `json:"active_this_cycle"`
+	InactiveThisCycle   int64 `json:"inactive_this_cycle"`
```

Comment:
Using the reasoning below, let's also please change these to `int`.

```suggestion
	Total               int `json:"total"`
	AddedThisCycle      int `json:"added_this_cycle"`
	PendingCancellation int `json:"pending_cancellation"`
	PendingInvitation   int `json:"pending_invitation"`
	ActiveThisCycle     int `json:"active_this_cycle"`
	InactiveThisCycle   int `json:"inactive_this_cycle"`
```